router/core/operation_planner.go

```diff
@@ -106,7 +109,7 @@ func (p *OperationPlanner) Plan(operation *ParsedOperation, clientInfo *ClientIn
 		if err != nil {
 			return nil, err
 		}
-		opContext.preparedPlan = &prepared
```

Comment:
Why was the type changed?

Reply:
preparePlan now returns pointer right away or nil in case of error
Jens asked for that